aws/lambda/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:3.9
+
+# Copy function code
+COPY lambda_function.py ${LAMBDA_TASK_ROOT}
```

Comment:
Out of curiosity, is this environment variable provided by AWS Lambda? 
Are we guaranteed to have this ENV?

Reply:
`LAMBDA_TASK_ROOT` is provided by Lambda as `/var/task`
(FYI: [Creating Lambda container images](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html))